Updated to allow arrays.

dcodadigital opened the pull request:
Hopefully useful but the idea of this pull request was to allow arrays to be added and inserted in similar format to that of Configure::write/read. 

Example when using an array name such as: name.anothername and so on:-
![pullrequest](https://cloud.githubusercontent.com/assets/14294908/10517759/441f5fbe-7356-11e5-8982-e0d2b375be8a.PNG)

dcodadigital committed:
Added array support

Added in support for setting arrays

dcodadigital committed:
Removed comment

dcodadigital committed:
Removed Configure

Not required. Originally added for testing purposes to compare
Configure::write against Setting::write.

dcodadigital committed:
Added serialize feature

Serializes arrays if used within Setting::write

bobmulder commented:
Hey,

I think the idea of your feature is great.
However some checks have failed. Please provide your feature with tests!

Bob

dcodadigital committed:
Tidy up plus test cases

General tidy up of code removing error suppression. Changes made
following phpunit tests as well as test cases added for testing
Setting::read/write arrays.

dcodadigital committed:
Strict Travis Conforms

bobmulder commented:
Closed because of #8
